modules/configuration/js/configuration_helper.js

```diff
@@ -39,31 +39,44 @@ $(function () {
 
         e.preventDefault();
 
-        var id = $(this).attr('name');
-        var button = this;
+        var options        = $(this).parent().parent().children().prop('options');
+        var selectedIndex  = $(this).parent().parent().children().prop('selectedIndex');
+        var selectedOption = options[selectedIndex].text;
+        var fieldName      = $(this).parent().parent().parent().parent().parent().children().attr('data-original-title');
+
+        if(
+            window.confirm(
```

Comment:
i'm having trouble getting my dev vm front-end working so i haven't seen what this dialogue box looks like. any chance you can add a screenshot to this PR? I'm wondering if using swal would work here so that the UI matches with the rest of LORIS

Reply:
Hi @zaliqarosli. Thank you for your comments. This is the way it looks in the UI.  

![image](https://user-images.githubusercontent.com/37309344/124595632-41944f80-de2f-11eb-9796-7fbe892cae13.png)

Reply:
@racostas could you try implementing a swal (sweetalert2) ?

Reply:
Hi @zaliqarosli, I'm not familiarized with the library but will try to give it a try.

Reply:
@racostas you can look for how it is being used elsewhere in loris and refer to the docs here https://sweetalert2.github.io/

Reply:
Hi @zaliqarosli, I'm still getting issues having the sweetalert2 properly working. It's seem the`LORIS configuration module` (the one we're modifying) * is not yet Reactified.

I'm trying Importing `sweetalert2` from the` project.js` page `/var/www/loris/modules/configuration/js/project.js` don't seems to work. Its not getting imported.

I think I should be missing something obvious but my experience with JS is not that extensive. If you can give me any insight will be very helpful. Thanks.

Reply:
@racostas i will check out your branch and play around. i'll let you know!

Reply:
hey @racostas it looks like sweetalert2 only works within React components which this js file is not. we do have sweetalert library in htdocs/vendor but we are trying to remove that. so I think until we reactify the configuration module, your implementation is the best option!

Reply:
@laemtl do you have solutions for us?

Reply:
@zaliqarosli @racostas  I have a working solution ready to be pushed. I just need permission to push on @racostas's branch.